.github/workflows/contracts.yml

```diff
@@ -196,6 +210,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Setup Credentials
```

Comment:
The solution you've provided here (without altering `contracts.yml`) seems sufficient (and even better, since this is a somehow more exotic case):

https://github.com/multiversx/mx-sdk-rust-contract-builder/issues/59#issuecomment-2115963239

Maybe we can promote and document that solution, and have `contracts.yml` untouched (at least, from the perspective of setting up credentials)?